lib/controllers/bucket.js

```diff
@@ -28,7 +29,7 @@ function generateContinuationToken(bucket, keyName, region) {
 function decipherContinuationToken(token) {
   const buf = Buffer.from(token, 'base64');
   if (buf.length < 8) return '';
-  const key = Buffer.alloc(8, 'S3RVER', 'utf8');
+  const key = Buffer.alloc(8, secretAccessKey, 'utf8');
```

Comment:
I think this can simply be a dummy string as long as it matches the reference above on line 15, since the token is not meant to be decoded by anyone but the S3 server. That would eliminate the reference to `secretAccessKey` here.

Reply:
Yeah this function is definitely missing some documentation explaining what's going on, my bad.

Basically, there's evidence that S3 uses a symmetric key to generate continuation tokens since the token length is proportional to the combined region, bucket name, and object key. The string I used here is arbitrary. If you'd like to have something more descriptive there instead, `'S3RVER_CTOKEN_KEY'` would probably be fine.